Source/Core/DolphinQt2/Config/Graphics/GraphicsWindow.cpp

```diff
@@ -27,15 +27,26 @@
 GraphicsWindow::GraphicsWindow(X11Utils::XRRConfiguration* xrr_config, MainWindow* parent)
     : QDialog(parent), m_xrr_config(xrr_config)
 {
-  g_Config.Refresh();
-  g_video_backend->InitBackendInfo();
+  // GraphicsWindow initialization is heavy due to dependencies on the graphics subsystem.
+  // To prevent blocking startup, we create the layout and children at first show time.
+}
+
+void GraphicsWindow::Initialize()
+{
+  if (!m_lazy_initialized)
```

Comment:
This can likely be inverted like:

```cpp
if (m_lazy_initialized)
  return;
```

which shortens the amount of code indented.